libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -245,11 +244,12 @@ class MapboxTripNotification constructor(
             routeProgress.bannerInstructions()?.let { bannerInstructions ->
                 if (isManeuverStateChanged(bannerInstructions)) {
                     updateManeuverImage(
-                            routeProgress.currentLegProgress()?.currentStepProgress()?.step()?.drivingSide()
-                                    ?: STEP_MANEUVER_MODIFIER_RIGHT
+                        routeProgress.currentLegProgress()?.currentStepProgress()?.step()?.drivingSide()
+                            ?: STEP_MANEUVER_MODIFIER_RIGHT
                     )
                 }
             }
+            setFreeDriveMode(false)
```

Comment:
The only change is to move this line from top to bottom inside the `let` block so the `let` block will always have a valid return value.

Reply:
Could we add a Unit / UI tests to avoid regressions in the future?

Reply:
Good point, working on it 👍